libs/math/tests/unit/activation_functions/sigmoid.cpp

```diff
@@ -144,23 +149,23 @@ TYPED_TEST(SigmoidTest, exact_values)
 TYPED_TEST(SigmoidTest, sigmoid_2x2)
 {
   using SizeType = fetch::math::SizeType;
+  using DataType = typename TypeParam::Type;
   TypeParam array1{{2, 2}};
 
-  array1.Set(SizeType{0}, SizeType{0}, typename TypeParam::Type(0.3));
-  array1.Set(SizeType{0}, SizeType{1}, typename TypeParam::Type(1.2));
-  array1.Set(SizeType{1}, SizeType{0}, typename TypeParam::Type(0.7));
-  array1.Set(SizeType{1}, SizeType{1}, typename TypeParam::Type(22));
+  array1.Set(SizeType{0}, SizeType{0}, fetch::math::Type<DataType>("0.3"));
+  array1.Set(SizeType{0}, SizeType{1}, fetch::math::Type<DataType>("1.2"));
+  array1.Set(SizeType{1}, SizeType{0}, fetch::math::Type<DataType>("0.7"));
+  array1.Set(SizeType{1}, SizeType{1}, DataType{22});
 
   TypeParam output = fetch::math::Sigmoid(array1);
 
   TypeParam numpy_output{{2, 2}};
 
-  numpy_output.Set(SizeType{0}, SizeType{0}, typename TypeParam::Type(0.57444252));
-  numpy_output.Set(SizeType{0}, SizeType{1}, typename TypeParam::Type(0.76852478));
-  numpy_output.Set(SizeType{1}, SizeType{0}, typename TypeParam::Type(0.66818777));
-  numpy_output.Set(SizeType{1}, SizeType{1}, typename TypeParam::Type(1));
-
-  ASSERT_TRUE(output.AllClose(numpy_output));
+  numpy_output.Set(SizeType{0}, SizeType{0}, fetch::math::Type<DataType>("0.57444252"));
+  numpy_output.Set(SizeType{0}, SizeType{1}, fetch::math::Type<DataType>("0.76852478"));
+  numpy_output.Set(SizeType{1}, SizeType{0}, fetch::math::Type<DataType>("0.66818777"));
+  numpy_output.Set(SizeType{1}, SizeType{1}, DataType{1});
+  ASSERT_TRUE(output.AllClose(numpy_output, fetch::math::function_tolerance<DataType>()));
```

Comment:
you've added the tolerance here - what tolerance was it using before?

Reply:
The default tolerance, "0.00001", which is too strict for FP<16,16>. I tried it without the tolerance and the other types passed